Fix the issue "Endless exception dialogues with illegal char in file path" (#7388)
JabRef/jabref: CHANGELOG.md

- We fixed an issue where the Harvard RTF exporter did not use the new authors formatter and therefore did not export "organization" authors correctly. [4508](https://github.com/JabRef/jabref/issues/4508)
- We fixed an issue where the field `urldate` was not exported to the corresponding fields `YearAccessed`, `MonthAccessed`, `DayAccessed` in MS Office XML [#7354](https://github.com/JabRef/jabref/issues/7354)
- We fixed an issue where the password for a shared SQL database was only remembered if it was the same as the username [#6869](https://github.com/JabRef/jabref/issues/6869)

> - We fixed an issue where endless exception dialogues with illegal char in file path. [#7381](https://github.com/JabRef/jabref/issues/7381)

Reviewer: ```suggestion
- We fixed an issue with endless exception dialogues appearing if the file path includes an illegal char. [#7381](https://github.com/JabRef/jabref/issues/7381)
```
